client/control/components/ColumnMenu.js

```diff
@@ -41,16 +41,14 @@ class ColumnMenu extends React.Component {
    * simply updates the component's column state,
    * and notifies the parent of what's selected
    *
-   * Note: it seems a little fragile to use the index for this,
-   * may be useful to switch to something more identifying.
    *
-   * @param  {Number}  columnIndex Integer, index of relevant column
-   * @param  {Boolean} isActive    Is the column now active?
+   * @param  {String}  columnValue  Value of the column ('name', 'endTime', etc)
```

Comment:
Yeah, the `value` name had come from the fact that was what the checkbox values were, but `slug` makes more sense! Should have switched it over earlier.